pyvista/core/objects.py

```diff
@@ -378,6 +378,29 @@ def plot(self, *args, **kwargs):
         """Plot the texture as image data by itself."""
         return self.to_image().plot(*args, **kwargs)
 
+    @property
+    def cube_map(self):
+        """Return ``True`` if cube mapping is enabled and ``False`` otherwise.
+
+        Is this texture a cube map, if so it needs 6 inputs, one for
+        each side of the cube. You must set this before connecting the
+        inputs.  The inputs must all have the same size, data type,
+        and depth.
+        """
+        return self.GetCubeMap()
+
+    @repeat.setter
+    def cube_map(self, flag):
+        """Enable cube mapping if ``flag`` is True, disable it otherwise."""
+        self.SetCubeMap(flag)
+
     def copy(self):
         """Make a copy of this texture."""
         return Texture(self.to_image().copy())
+
+    def to_skybox(self):
+        """Return the texture as a ``vtkSkyBox`` if cube mapping is supported."""
```

Comment:
If there's a stronger statement here than "enabled", then I think we should somehow verify that the right inputs were passed for a cube mapping. Or catch VTK's error if it raises one.
```suggestion
        """Return the texture as a ``vtkSkybox`` if cube mapping is enabled."""
```

Reply:
Agreed.